### ISSUE TITLE
[1.1.2(22)] Samsung Galaxy Tab S2/5.0.2, Samsung Galaxy S4/5.0.1- User is not able to export the lomotif.

### ISSUE THREAD

**Issue by testershub** [Lomotif/lomotif-android-issues#197]
Frequency: Every time

Action: 
1. Launch the application. 
2. Tap on "Start" button.
3. Tap on "Motif" and select any video, for example "Friendship".
4: Tap on "Next" icon and select any music.
5: Tap on "Next" icon and tap on "Export" icon.
6: Pop up message "Are you ready.." will be displayed, tap on "YES" option.
7: Status bar will be displayed and user will be navigated to "Export" screen(Show it to the world).
8: Tap on device back button (hard key).
9: Again tap on "Export" icon.
10: Check if user is able to export the lomotif.

Expected Result: User should be able to export the lomotif.

Actual Result: User is not able to export the lomotif.

Refer the following video link for more information:
http://www.screencast.com/t/5YNf7Ns1A

**Comment by pallavipillai** [comment]
I am able to reproduce the issue for build [1.1.2(22)] on Nexus 6/Android Version 6.0.1

**Comment by vysohanych** [comment]
I can reproduce it on build 1.1.2.(22) on Samsung Galaxy Tab S 10.5 Android 4.4.2.

**Comment by Dipamit** [comment]
I am able to reproduce this issue on Xiaomi Redmi Note 3/5.1.1 for build[1.1.2(22)]

**Comment by xiuie** [comment]
@testershub You pressed "Export Video" twice. Did the Lomotif video save to your gallery for the FIRST time?

**Comment by testershub** [comment]
@xiuie Yes it got saved to the gallery for the first time. But when I tap on the saved video from the landing screen and try to export the same again, I am able to export the same again. But when I tap on back button from the "Share" screen I am navigated back to the "Create" screen but I am not able to export this saved video again.

**Comment by hiteshbheda** [comment]
@xiuie I can reproduce this issue for build [1.1.2 (22)] on Asus Zenfone Go 4.5 / Android 5.1
